### PR TITLE
chore: March 2021 patch updates for Windows

### PR DESCRIPTION
**Pull request by jsturtevant**
**Reason for Change**:

March patch tuesday

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -195,9 +195,9 @@ function Install-WindowsPatches {
             # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
-            # KB4598230 contains January 12, 2021 cumulative updates for Windows Server 2019
-            # https://www.catalog.update.microsoft.com/Search.aspx?q=KB4598230
-            $patchUrls = @("http://download.windowsupdate.com/d/msdownload/update/software/secu/2021/02/windows10.0-kb4601345-x64_6dfee9d6f028678d7988eb35cd5c0867bf96e4c6.msu")
+            # KB4598230 contains March 9, 2021 cumulative updates for Windows Server 2019
```
Comment (AbelHu) [inline vhd/packer/configure-windows-vhd.ps1:198]: Do you need to update the base image to 17763.1757.2102060435?

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:198]: I was going to add this comment too.
I don't think it *needs* to be done but would be nice to have.

Comment (jsturtevant) [reply vhd/packer/configure-windows-vhd.ps1:198]: done

**Review by marosset — APPROVED** [review]
/lgtm

**Pull request merged**